src/java/org/apache/cassandra/cql3/statements/schema/CopyTableStatement.java

```diff
@@ -229,12 +234,37 @@ public void validate(ClientState state)
         validateDefaultTimeToLive(attrs.asNewTableParams());
     }
 
+    private void cloneIndexesIfNeeded(TableMetadata.Builder builder, TableMetadata sourceTableMeta, KeyspaceMetadata targetKeyspaceMeta)
+    {
+        if (createLikeOption == CreateLikeOption.INDEX)
```

Comment:
I would do 

````
if (createLikeOption != CreateLikeOption.INDEX)
    return;

...
````

By doing that, we make the number of indentation smaller etc.